pkg/applicationserver/io/web/webhooks.go

```diff
@@ -129,6 +138,7 @@ var errQueueFull = errors.DefineResourceExhausted("queue_full", "the queue is fu
 func (s *QueuedSink) Process(req *http.Request) error {
 	select {
 	case s.Queue <- req:
+		webhookMetrics.webhookQueue.Inc()
 		return nil
 	default:
 		return errQueueFull.New()
```

Comment:
Could we please count the number of dropped messages as well ? It's an important metric since it means we need to add more workers due to slow hosts (there is a scalability issue about this, but the metrics are still useful for this implementation as well).